src/pages/signin/PasswordForm.js

```diff
@@ -140,7 +159,7 @@ class PasswordForm extends React.Component {
                         isLoading={this.props.account.loading}
                         onPress={this.validateAndSubmitForm}
                     />
-                    <ChangeExpensifyLoginLink />
+                    {this.props.isVisible && <ChangeExpensifyLoginLink />}
```

Comment:
Why?

Reply:
`ONYXKEYS.CREDENTIALS` could be null so it was causing a runtime issue in `ChangeExpensifyLoginLink`. Now, I have made it optional and passed a default value so we don't need it.